src/platforms/unity/index.mdx

```diff
@@ -1,4 +1,8 @@
-<PlatformContent includePath="getting-started-primer" />
+<Note>
```

Comment:
You don't need an index page. See previous notes (simply populate the directories with the appropriate content).

Reply:
There was a lot more content in there given the custom Window.
We'll change the SDK next to add the programatic Init approach adn get back to docs to take more of the common content